Earthfile

```diff
@@ -42,23 +46,28 @@ run-coverage:
     DO rust-udc+CARGO --args="llvm-cov report --html --output-dir /tmp/coverage/"
     SAVE ARTIFACT /tmp/coverage/*
 
+# build runs tests and then creates a release build
 build:
     BUILD +run-tests
     BUILD +build-release
 
+# build-create creates a .crate file for distribution of source code
```

Comment:
Typo: `create` instead of `crate` (I do this 90% of the time I try to write crate...).

Given the display context:
```
  +build
      build runs tests and then creates a release build
```

We should consider eliding the target name, e.g.

```
  +build
      runs tests and then creates a release build
```

I think that works in the source too:
```
# runs tests and then creates a release build
build:
```

The rule would be "the doc comment should be a sentence fragment suitable to be appended to the target name".

Reply:
I've fixed the `Typo: create instead of crate` with another push :)

Reply:
If there is a style guide which we can follow then we should do so!